packages/api-client/tests/variable.spec.ts

```diff
@@ -110,6 +110,8 @@ describe('Get Variable Tests', () => {
     expect(variable.data.versions.length).toBe(1)
     expect(variable.data.versions[0].value).toBe('Variable 2 value')
     expect(variable.data.versions[0].environmentId).toBe(environment.id)
```

Comment:
I don't think this would be present anymore. And if its still being returned, then you might have missed something.